conf/logback-test.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+
+    <appender name="FILE" class="ch.qos.logback.core.FileAppender">
+        <file>logs/income-tax-view-change-frontend.log</file>
```

Comment:
`logs/income-tax-view-change-frontend.log` should be `logs/income-tax-view-change.log`